server/src/rbx_snapshot.rs

```diff
@@ -348,9 +374,29 @@ fn snapshot_imfs_directory<'source>(
         }
     }
 
+    if let Some(meta_ignore_instances) = meta.ignore_unknown_instances {
+        snapshot.metadata.ignore_unknown_instances = meta_ignore_instances;
+    }
+
     Ok(Some(snapshot))
 }
 
+#[derive(Debug, Default, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+struct InitMetaJson {
```

Comment:
I don't think we need the `Json` part of this struct name.

Since we don't serialize the struct, we might be able to drop the `Serialize` derive and all of the `skip_serializing_if` attributes as well.